pkgs/servers/mail/mailman/web.nix

```
@@ -1,37 +1,38 @@
-{ stdenv, python, hyperkitty, postorius, buildPythonPackage
-, serverEMail ? "postmaster@example.org"
-, archiverKey ? "SecretArchiverAPIKey"
-, allowedHosts ? []
+{ buildPythonPackage, lib, fetchgit
```

follow up to my other suggestion
```suggestion
{ buildPythonPackage, isPy3k, lib, fetchgit
```